src/content/blog/international-crypto-card.md

```diff
@@ -0,0 +1,101 @@
+
+Some people compromise security for convenience and use 3rd party **custodial wallets**. It provides convenience, yet in essence, they don’t own the crypto anymore. The **“not your keys – not your coins”** mantra refers to the recommended practice of storing digital assets in non-custodial wallets. Such a wallet is merely an interface to the blockchain – the ledger with the immutable balance of each and every user. Accessing funds on a blockchain, with your own keys, without any 3rd party custodian involved, is a secure way to transact with digital assets.
+
+## Seamless International Transactions with Crypto. Possible
+
+Crypto payment cards provide one of the best options to combine crypto utility with daily payments. But there is a caveat: not all crypto payment cards are the same:
+
+**Crypto credit or debit cards linked to non-custodial wallets streamline the process of using the crypto, yet still keep your crypto secure and in your ownership.**
+
+Technically, the service provider pulls the funds through a custodial channel, where the crypto-to-fiat exchange takes place. On the other end, the channel is integrated with the banking system and uses the same payment processors and banking accounts as the regular banking payment cards.
+
+SORA Card works both ways. It is a convenient way to buy digital assets with fiat currency and tap into the world of seamless cross-border transfers and Decentralized Finance (DeFi) that crypto can provide. Again, after processing through the custodial channel, the funds are settled on the blockchain, with the only access to them via the owner’s keys.
+
+![Flowchart](/blog/international-crypto-card/image1.jpg)
+
+As SORA Card is certified by major financial network, Mastercard, it is taking that convenience one step further by allowing users to tap into their crypto wallets while traveling or shopping online.
+
+## International Crypto Card
+
+It is not surprising that payment cards were chosen as a bridging solution. Introduced more than 50 years ago, credit and debit cards have since become an everyday part of our lives. With penetration above 60% in most countries, bank cards are accepted almost everywhere and are a convenient and secure payment method for daily use, travel, and online shopping. Merchants are eager to integrate cards into their payment terminals because of the vast international clientele it brings them.
+
+Card processing technology is constantly evolving, adding additional security mechanisms such as advanced encryption of messages, secure chips, and others. Combining the payment industry’s security mechanisms with decentralized (crypto) security practices is an essential step for the evolution of the latter.
+
+SORA Card aims to combine the best features of the two systems and merge them into a new money management paradigm:
+
+![Card Benefits](/blog/international-crypto-card/sora card-benefits.jpg)
```

Comment:
`space` is not allowed in the image name, should be `-`: 

```
![Card Benefits](/blog/international-crypto-card/sora-card-benefits.jpg)
```

Reply:
resolved 👍